doc/classes/CanvasLayer.xml

```diff
@@ -53,6 +53,7 @@
 		</member>
 		<member name="layer" type="int" setter="set_layer" getter="get_layer" default="1">
 			Layer index for draw order. Lower values are drawn behind higher values.
+			[b]Note:[/b] If multiple CanvasLayers have the same layer index, [CanvasItem]-children of one CanvasLayer are drawn behind the [CanvasItem]-children of the other CanvasLayer. Which CanvasLayer is drawn in front is non-deterministic.
```

Comment:
Is it really non-deterministic, or is it in scene tree order of the CanvasLayers?

Reply:
It is not only the scene tree order. You can see that in the Video of the linked issue.

Reply:
See also #69952 which made canvas layer ordering deterministic in 3.x.